launchcontainer/launchcontainer.py

```diff
@@ -169,13 +167,14 @@ def wharf_url(self, force=False):
                 "ENV_TOKENS['LAUNCHCONTAINER_API_CONF']",
                 settings.ENV_TOKENS.get('LAUNCHCONTAINER_API_CONF', {}).get('default')
             ),
-            # Fallback to the default.
-            (
-                "Default", DEFAULT_WHARF_URL
-            )
         )
 
-        url = next((x[1] for x in urls if is_valid(x[1])))
+        try:
+            url = next((x[1] for x in urls if is_valid(x[1])))
+        except StopIteration:
+            raise ImproperlyConfigured("No Virtual Labs URL was found, "
+                                       "please contact your site administrator.")
+
         if not url:
```

Comment:
Ah, good catch. Now that I raise the `StopIteration`, this won't happen. Will fix. Good eye! 👁